src/renderer/component/common/icon.jsx

```diff
@@ -19,8 +19,10 @@ class IconComponent extends React.PureComponent<Props> {
     const Icon = FeatherIcons[icon];
 
     let color;
-    if (icon === icons.HEART || icon === icons.FEATURED) {
+    if (icon === icons.TRASH || icon === icons.FEATURED) {
```

Comment:
This may not scale too well, also icons may render different colors based on context

Reply:
👍 I'll create a `color` prop and use that instead. Then any icon can have a color.